source/reference/minio-mc-admin/mc-admin-rebalance.rst

```diff
@@ -135,6 +136,12 @@ Syntax
 
    Ends an in-progress rebalance job on the specified deployment.
 
+   .. admonition:: Stopping a rebalance job on previous versions of MinIO may cause data loss
+      :class: warning
+
+      A bug in MinIO prior to :minio-release:`RELEASE.2024-08-17T01-24-54Z` can overwrite objects while stopping a in-progress rebalance operation. 
+      Interrupting rebalance on these older versions may result in data loss.
+
```

Comment:
Is there a usual way we reference things like this? What should we say about the now fixed bug?